lib/generators/index.js

```diff
@@ -12,7 +12,7 @@ module.exports = function(generatorConfig) {
 
         default:
             // Look for a custom generator
-            var CustomGenerator = require("custom-generators/" + generatorConfig.generator);
+            var CustomGenerator = require("../../custom-generators/" + generatorConfig.generator + ".js");
```

Comment:
Correct for the `../../`, on the other hand, `+ ".js"` is not required.
I'll change that.